### DISCUSSION TITLE
Option to move mouse cursor to middle of selected window when cursor is used for selection.

### DISCUSSION THREAD

**Discussion by christianmagill** [rxhanson/Charmstone-Community#24] [Ideas]
On large monitors, it can be quite a distance to travel from one side of the screen to another.

**Comment by rxhanson** [comment]
Unfortunately, since Charmstone is sandboxed to be on the App Store, I can't use the window API to get the locations & dimensions of windows. If I roll out a non-sandboxed version of Charmstone outside of the App Store, then I'll look at adding this in. 

I am planning on adding cursor actions like this to Rectangle Pro in an upcoming release (move cursor to center of frontmost window, etc).

**Comment by rokn** [comment]
Yeah this would help a lot as I use a automatic focus switcher which switches the focus to the app under the mouse, which doesn't work well with Charmstone as it's unfocusing the chosen window from Charmstone when used with multiple displays/not fullscreen apps

**Comment by rphaedrus** [comment]
This would be great even if it just put it anywhere on the app, especially when using multiple monitors.  I wonder if there could be a lighthouse integration where if you're running both, lighthouse moves the cursor when charmstone focuses a window?

IMO, it would be nice whether one is using keyboard or mouse to focus the app.

**Comment by horeshas** [comment]
This feature is badly needed on large monitors!
